web/gds-admin-ui/src/helpers/api/apiCore.js

```diff
@@ -188,17 +215,21 @@ class APICore {
             this.setLoggedInUser({ token, ...user, ...modifiedUser });
         }
     };
+
+    deleteUserSession = () => {
+        sessionStorage.removeItem(AUTH_SESSION_KEY)
+        window.location.href = '/login'
+    }
```

Comment:
Should we also call `setAuthorization(null)` to remove the access token from the default common headers?

Reply:
alright